lib/streamlit/bootstrap.py

```diff
@@ -21,6 +21,7 @@
 import tornado.ioloop
 from streamlit.git_util import GitRepo, MIN_GIT_VERSION
 
+from streamlit import version
```

Comment:
Feel free to ignore this since the imports are already not ordered alphabetically (although they do seem to be mostly-sorted), but it'd be nice if imports within a grouping were shuffled around so that they were alphabetical

Reply:
updated